rancher/distros-test-framework
# add Kubeconfig

#### Proposed Changes ####

- https://confluence.suse.com/display/RKE2K3S/Add+existent+KubeConfig+on+test+framework
- https://github.com/rancher/qa-tasks/issues/862 




#### Types of Changes ####



#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

https://github.com/rancher/distros-test-framework/pull/128#issuecomment-2231887499 

2. Verify code lint; we should not have errors.

ok 

3. Update the documentation if needed.

ok 

4. Update makefile and docker run if adding new tests.

n/a

5. Run your tests at least 4 times with all configurations needed and possible.

ok 
6. If needed test with different os types.

ok 
#### Linked Issues ####



#### Further Comments ####
